### DISCUSSION TITLE
TextTrimming

### DISCUSSION THREAD

**Discussion by xiaoliangde** [AvaloniaUI/Avalonia#15408] [Q&A]
### Describe the bug

It can'be correct when paren is StackPane(by Orientation="Horizontal").

### To Reproduce

It can'be correct when paren is StackPane(by Orientation="Horizontal")

### Avalonia version

11.0.9

**Comment by rabbitism** [comment]
Horizontal StackPanel width is infinite. Text will never be trimmed in it.
